frontend/vite.config.js

```diff
@@ -34,6 +34,11 @@ export default defineConfig({
       { find: "reducers", replacement: resolve(projectRootDir, "./src/reducers") },
     ],
   },
+  server: {
+    watch: {
+      usePolling: true
```

Comment:
when I was using wsl for some reason the hot reload wasn't working, this fixed it though https://github.com/vitejs/vite/issues/1153

Reply:
@leonardo-fan this suffers a performance hit, you should move your files to the linux file system instead. It will save you many more headaches with WSL. Trust me.

Reply:
ooft i see - will do